Event and city views

Lindayezou opened the pull request:
Renders random gif to page.

Lindayezou committed:
Sets up structure for temp city and event logic

Sets up folders for events and city
Sets up working events logic

 Changes to be committed:
	modified:   city/city.html
	modified:   event/event.html
	modified:   event/event.js
	modified:   services.js
	modified:   style.css
	new file:   ../server/data/tempCityData.js
	new file:   ../server/data/tempEventData.js
	modified:   ../server/routes.js
	new file:   ../server/tempCity/tempCityController.js
	new file:   ../server/tempEvents/tempEventsController.js

Lindayezou committed:
Adds logic for working temp City view and cleans up events view logic

 Changes to be committed:
	modified:   city/city.html
	modified:   city/city.js
	modified:   event/event.html
	modified:   services.js
	modified:   ../server/data/tempCityData.js
	modified:   ../server/routes.js
	modified:   ../server/tempCity/tempCityController.js
	modified:   ../server/tempEvents/tempEventsController.js

Lindayezou committed:
Implements sockets for gif/description loading

 Changes to be committed:
	modified:   event/event.js
	modified:   ../server/server.js

Lindayezou committed:
Implements socket logic for city gif load

Changes name for event socket to make it unique to function

 Changes to be committed:
	modified:   city/city.js
	modified:   event/event.js
	modified:   ../server/server.js

Lindayezou committed:
Cleans up html pages for event and city

 Changes to be committed:
	modified:   city/city.html
	modified:   event/event.html

Lindayezou committed:
Adds style to event and city

 Changes to be committed:
	modified:   city/city.html
	modified:   event/event.html

Lindayezou committed:
Adds proper gifs and descriptions and refactors views to match

 Changes to be committed:
	modified:   city/city.html
	modified:   event/event.html
	modified:   ../server/data/tempCityData.js
	modified:   ../server/data/tempEventData.js

Lindayezou committed:
Fixes type in city data file - url string

 Changes to be committed:
	modified:   ../server/data/tempCityData.js

chououtside commented on server/tempCity/tempCityController.js:
```diff
@@ -0,0 +1,11 @@
+var Q = require('q');
```
Doesn't use Q, unnecessary

chououtside commented on server/tempCity/tempCityController.js:
```diff
@@ -0,0 +1,11 @@
+var tempCityData = require('../data/tempCityData.js');
+
+  module.exports.getRandomURL= function(req, res, next) {
+    var x = Math.floor(Math.random() * tempCityData.length)
```
semicolon

chououtside commented on server/tempEvents/tempEventsController.js:
```diff
@@ -0,0 +1,11 @@
+var Q = require('q');
```
dont use q

Lindayezou committed:
Cleans up code and made edits base on comments

 Changes to be committed:
	modified:   city/city.js
	modified:   event/event.js
        modified:   ../server/tempCity/tempCityController.js
	modified:   ../server/tempEvents/tempEventsController.js

chououtside commented:
Sockets Not Working 100% of the time to synchronize all views.
